react/features/base/connection/actions.native.js

```diff
@@ -275,10 +275,17 @@ function _constructOptions(state) {
     // redux store.
     const options = _.cloneDeep(state['features/base/config']);
 
-    // Normalize the BOSH URL.
-    let { bosh } = options;
+    let { bosh, websocket } = options;
```

Comment:
in connection.js we are getting rid of .bosh config, is there a reason to have it here?

Reply:
Aaa we remove the passing it to ljm, but we still have it in config.js, is this right?

Reply:
Correct!